pom.xml

```diff
@@ -1,331 +1,307 @@
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.scijava</groupId>
+		<artifactId>pom-scijava</artifactId>
+		<version>22.1.0</version>
+		<relativePath />
+	</parent>
+
+	<artifactId>ThunderSTORM</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
+
+	<name>ThunderSTORM</name>
+	<description>ThunderSTORM: a comprehensive ImageJ plugin for PALM and STORM data analysis and super-resolution imaging</description>
+	<url>https://github.com/zitmen/thunderstorm</url>
+	<inceptionYear>2014</inceptionYear>
+
+	<organization>
+		<name>SciJava</name>
+		<url>http://www.scijava.org/</url>
+	</organization>
+
+	<licenses>
+		<license>
+			<name>MIT License</name>
+			<url>http://choosealicense.com/licenses/mit/</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<id>zitmen</id>
+			<name>Martin Ovesný</name>
+			<email>zitmen@gmail.com</email>
+			<organization>Institute of Cellular Biology and Pathology, First Faculty of Medicine, Charles University in Prague</organization>
+			<organizationUrl>http://lge.lf1.cuni.cz/</organizationUrl>
+			<timezone>+1</timezone>
+		</developer>
+		<developer>
+			<id>pepa_borkovec</id>
+			<name>Josef Borkovec</name>
+			<email>vmodrosedem@gmail.com</email>
+			<organization>Institute of Cellular Biology and Pathology, First Faculty of Medicine, Charles University in Prague</organization>
+			<organizationUrl>http://lge.lf1.cuni.cz/</organizationUrl>
+			<timezone>+1</timezone>
+		</developer>
+	</developers>
+	<contributors>
+		<!-- NB: Need a least one element to override the parent. See: https://issues.apache.org/jira/browse/MNG-5220 -->
+		<contributor>
+			<name>None</name>
+		</contributor>
+	</contributors>
+
+	<properties>
+		<package-name>ThunderSTORM</package-name>
+		<license.licenseName>mit</license.licenseName>
+		<license.copyrightOwners>Martin Ovesný</license.copyrightOwners>
+
+		<kotlin.version>1.1.2-4</kotlin.version>
+		<kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
+		<kotlin.compiler.incremental>true</kotlin.compiler.incremental>
+
+		<protobuf-java.version>2.5.0</protobuf-java.version>
+		<gson.version>2.7</gson.version>
+
+		<!-- NB: Deploy releases to the ImageJ Maven repository. -->
+		<releaseProfiles>deploy-to-imagej</releaseProfiles>
+	</properties>
+
+	<mailingLists>
+		<mailingList>
+			<name>ImageJ Forum</name>
+			<archive>http://forum.imagej.net/</archive>
+		</mailingList>
+	</mailingLists>
+
+	<scm>
+		<connection>scm:git:git://github.com/zitmen/thunderstorm</connection>
+		<developerConnection>scm:git:git@github.com:zitmen/thunderstorm</developerConnection>
+		<tag>HEAD</tag>
+		<url>https://github.com/zitmen/thunderstorm</url>
+	</scm>
+	<issueManagement>
+		<system>GitHub Issues</system>
+		<url>https://github.com/zitmen/thunderstorm/issues</url>
+	</issueManagement>
+	<ciManagement>
+		<system>Travis</system>
+		<url>https://travis-ci.org/zitmen/thunderstorm</url>
```

Comment:
travis-ci.org is migrating to travis-ci._com_
It's recommended for new projects to directly use travis-ci.com, the `travisify.sh` script works with both org and com since a few days, thanks to @ctrueden.